
Erhm - on PPlus I do not want the non-compileable in this config
macintosh/nvram driver ...

  - Rene Rebe <rene@exactcode.de>

--- linux-2.4.25/drivers/macintosh/Makefile.orig	2002-11-29 00:53:13.000000000 +0100
+++ linux-2.4.25/drivers/macintosh/Makefile	2004-04-19 21:51:26.000000000 +0200
@@ -28,7 +28,7 @@
 
 obj-$(CONFIG_PMAC_PBOOK)	+= mediabay.o
 obj-$(CONFIG_MAC_SERIAL)	+= macserial.o
-ifneq ($(CONFIG_MAC),y)
+ifeq ($(CONFIG_PMAC),y)
   obj-$(CONFIG_NVRAM)		+= nvram.o
 endif
 obj-$(CONFIG_MAC_HID)		+= mac_hid.o
